config/sharedSettings.php

```diff
@@ -10,6 +10,7 @@
                 'maxReservationTime' => env('PROJECT_RESERVATION_TIME', 30)
             ]
         ],
+        'default_role' => 'standard'
```

Comment:
can not merge because of this, there would be a conflict. Please update PR with latest code master